src/components/FuelUsed/FuelForm.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export default function FuelForm () {
+    return (
+        <div>
+            {/* Replace this placeholder route */}
+            <form action="/api/form" method="POST">
```

Comment:
Instead of using an action, use the onSubmit function for forms. It'll still trigger from the submit button.

Then you can add a handler function which is executed when onSubmit triggers.

Ex: `<form onSubmit={(e) => handleFormSubmit(e)}> ... </form>`

and 

` const handleFormSubmit = (e: React.FormEvent) => { <your function stuff> }` (put this before the return of the component)